README.md

```diff
@@ -70,16 +70,15 @@ npm install --save single-spa-angular
 
 Then create `main.single-spa.ts` with the following content:
 ```typescript
-import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
-import {ApplicationRef} from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { ApplicationRef } from '@angular/core';
 import singleSpaAngular from 'single-spa-angular';
-import mainModule from './main-module.ts';
-import {Router} from '@angular/router';
+import { Router } from '@angular/router';
+import { AppModule } From './app/app.module';
 
 export default singleSpaAngular({
+  bootstrapFunction: () => platformBrowserDynamic().bootstrapModule(AppModule),
```

Comment:
This is what makes the bundle size go down -- Angular can statically analyze this code and replace platformBrowserDynamic with platformBrowser, which is smaller and intended for production builds.